test/e2e/test_cluster_configs/sgx.json

```diff
@@ -0,0 +1,42 @@
+{
+	"env": {
+		"REGION_OPTIONS": "westeurope"
+	},
+	"apiModel": {
+        "apiVersion": "vlabs",
+        "properties": {
+          "orchestratorProfile": {
+            "orchestratorType": "Kubernetes"
+          },
+          "masterProfile": {
+            "count": 1,
+            "vmSize": "Standard_D2s_v3",
+            "dnsPrefix": ""
+          },
+          "agentPoolProfiles": [
+            {
+              "name": "agentpool",
+              "count": 1,
+              "availabilityProfile": "VirtualMachineScaleSets",
+              "distro": "aks-ubuntu-18.04",
+              "vmSize": "Standard_DC4s",
```

Comment:
@vtikoo in order to better manage quota on our test subs let's change this to `Standard_DC2s`. I imagine the hello world test job will be equivalently covered with a 2 core node, right?